src/regressor.jl

```diff
@@ -42,34 +45,41 @@ ncols(X) = Tables.columns(X) |> Tables.columnnames |> length
 
 A private method that returns the shape of the input and output of the model for given
 data `X` and `y`.
-
 """
 shape(model::MultitargetNeuralNetworkRegressor, X, y) = (ncols(X), ncols(y))
 
 build(model::MultitargetNeuralNetworkRegressor, rng, shape) =
     build(model.builder, rng, shape...)
 
-function fitresult(model::MultitargetNeuralNetworkRegressor, chain, y)
+function fitresult(
+    model::MultitargetNeuralNetworkRegressor,
+    chain,
+    y,
+    ordinal_mappings=nothing,
+    embedding_matrices=nothing,
+)
     if y isa Matrix
         target_column_names = nothing
     else
         target_column_names = Tables.schema(y).names
     end
-    return (chain, target_column_names)
+    return (chain, target_column_names, ordinal_mappings, embedding_matrices)
 end
 
 function MLJModelInterface.predict(model::MultitargetNeuralNetworkRegressor,
     fitresult, Xnew)
-    chain, target_column_names = fitresult
+    chain, target_column_names, ordinal_mappings, _ = fitresult
+    Xnew = ordinal_encoder_transform(Xnew, ordinal_mappings)
     X = reformat(Xnew)
     ypred = [chain(values.(tomat(X[:, i])))
              for i in 1:size(X, 2)]
-    output = isnothing(target_column_names) ? permutedims(reduce(hcat, ypred)) :
-        MLJModelInterface.table(reduce(hcat, ypred)', names=target_column_names)
+    output =
+        isnothing(target_column_names) ? permutedims(reduce(hcat, ypred)) :
+        MLJModelInterface.table(reduce(hcat, ypred)', names = target_column_names)
     return output
 end
 
 MLJModelInterface.metadata_model(MultitargetNeuralNetworkRegressor,
-    input=Union{AbstractMatrix{Continuous},Table(Continuous)},
-    target=Union{AbstractMatrix{Continuous}, Table(Continuous)},
-    path="MLJFlux.MultitargetNeuralNetworkRegressor")
+    input = Union{AbstractMatrix{Continuous}, Table(Finite)},
```

Comment:
Looks wrong to me.  Input can still include `Continuous` features as well as the new `Finite` ones. If I assume matrix input continues to be restricted to continuous scitype, then:

```suggestion
    input = Union{AbstractMatrix{Union{Continuous}, Table(Continuous,Finite)},
```

Reply:
Indeed, thanks for pointing out. But why the `Union` in `Union{Continuous}`?